src/js/angular/import/directives/import-resource-tree.directive.js

```diff
@@ -180,6 +180,24 @@ function importResourceTreeDirective($timeout, ImportContextService) {
                 $scope.onEditResource({resource});
             };
 
+            $scope.toggleSelectAll = () => {
+                const allSelected = $scope.displayResources.every((resource) => resource.selected);
```

Comment:
I don't like these operations with the displayResources property. I know that you are just following the pattern, but it's time for you to start thinking about the code quality practices we are talking constantly. So, let's do this right, can we? 
First. Create another model which wraps the `ImportResourceTreeElement[]` list returned by the `resources.toList()`. Then implement methods for all such needs inside. Then update references where needed and reuse those methods, thus effectively abstracting the internal implementation of the tree elements list.